### ISSUE TITLE
When an analysis is not waiting but is running loading message should say "Analysis XX running..."

### ISSUE THREAD

**Issue by saleiva** [CartoDB/cartodb#8926]
Right now it just says Analysis waiting

It's important that we differentiate when it's running and when it's enqued/waiting on the UI.

**Comment by nobuti** [comment]
From the UI perspective, it's already tracked properly:

![screen shot 2016-08-01 at 16 55 13](https://cloud.githubusercontent.com/assets/1366843/17298327/c92bb6ba-5809-11e6-8ce1-de3aeb5fdf27.png)

but the thing is that it's really difficult to make a job enqueued, almost one millisecond after that, the job was ready, so the notification changes to complete.
